app/services/gtfs_graph.rb

```diff
@@ -15,34 +15,61 @@ def initialize(filename, feed, feed_version)
     @onestop_id_to_entity = {}
   end
 
-  def create_change_osr(import_level=0)
-    raise ArgumentError.new('import_level must be 0, 1, or 2.') unless (0..2).include?(import_level)
-    log "Load GTFS"
+  def create_change_osr(import_level)
```

Comment:
Unused method argument - `import_level`. If it's necessary, use `_` or `_import_level` as an argument name to indicate that it won't be used. You can also write as `create_change_osr(*)` if you want the method to accept any arguments but don't care about them.